pallets/gear-program/src/migrations/allocations.rs

```diff
@@ -251,24 +264,18 @@ mod test {
             assert!(!w.is_zero());
             MigrateAllocations::<Test>::post_upgrade(state).unwrap();
 
-            if let Program::Active(p) = ProgramStorage::<Test>::get(active_program_id).unwrap() {
-                assert_eq!(
-                    p.allocations.to_vec(),
-                    [
-                        WasmPage::from(1)..=WasmPage::from(5),
-                        WasmPage::from(101)..=WasmPage::from(102)
-                    ]
-                );
-                assert_eq!(
-                    p.pages_with_data.to_vec(),
-                    [
-                        GearPage::from(4)..=GearPage::from(8),
-                        GearPage::from(400)..=GearPage::from(401)
-                    ]
-                );
-            } else {
+            let allocations = AllocationsStorage::<Test>::get(active_program_id).unwrap();
```

Comment:
`allocations_tree_len` assertion would be nice

Reply:
done